dod-advana/gamechanger-web
# Feature/UOT-108912 EDA Similar Docs Section

## Description

- make initial version of similar docs section to EDA Contract Detail Page (shown as an accordion named Similar Documents)
- fix total doc count on EDA Contract Detail Page

## Related Issue/Ticket

https://confluence.di2e.net/display/UOT/UOT-108912+EDA+Add+Initial+Similar+Docs+Section


## Testing instructions

- make sure you have EDA .env configs (can ask Aaron)
- navigate to /#/contractsearch
- make any search
- click on the Contract button on a card (may need to try different results, some cards do not have metadata included)
- check for the Similar Documents dropdown accordion
